test/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { Connection } from '../src/db/Connection';
+
+const dbUrl = process.env.DATABASE_URL || 'mongodb://localhost:27017/measure-repository';
+
+const createTestResource = async (data: any, resourceType: string) => {
+  const collection = Connection.db.collection(resourceType);
+  await collection.insertOne(data);
+  return { id: data.id };
+};
```

Comment:
Does this work?

```suggestion
const createTestResource = async (data: fhir4.FhirResource, resourceType: FhirResourceType) => {
  const collection = Connection.db.collection<fhir4.FhirResource>(resourceType);
  await collection.insertOne(data);
  return { id: data.id };
};
```

Remember to not use `any` unless you really have to. The above seems to work for me without running into those crazy issues we got yesterday (that must've been unique to `.find`